server.ts

```diff
@@ -1,7 +1,9 @@
-import { createRequestHandler } from "@netlify/remix-adapter";
 import * as build from "@remix-run/dev/server-build";
+import { createRequestHandler } from "@netlify/remix-adapter";
 
-export const handler = createRequestHandler({
+const handler = createRequestHandler({
   build,
   mode: process.env.NODE_ENV,
 });
+
+export default handler;
```

Comment:
It doesn't recognise it as a v2 function if we export it directly, so we assign to a var and export that